flink/src/test/java/org/apache/iceberg/flink/SimpleDataUtil.java

```diff
@@ -259,4 +265,37 @@ public static StructLikeSet actualRowSet(Table table, Long snapshotId, String...
 
     return dataFiles;
   }
+
+  public static Map<Long, List<DataFile>> snapshotToDataFiles(
+      Table table)
+      throws IOException {
+    table.refresh();
+    Map<Long, List<DataFile>> res = Maps.newHashMap();
```

Comment:
nit: I believe the style is to use more complete word
res -> result
mf -> manifestFile

Reply:
Done